### ISSUE TITLE
Problem Executing alevin_qc.r

### ISSUE THREAD

**Issue by StevenWingett** [nf-core/scrnaseq#100]
## Description of the bug

The scrna pipeline runs, but appears to fail when running the alevin_qc.r plot.

## Steps to reproduce

Steps to reproduce the behaviour:

1. Command line:
nextflow run nf-core/scrnaseq -r 1.1.0 --input './SLX-21534*_{1,2}.fq.gz' --fasta fly.fa --gtf /public/genomics/species_references/nextflow/Genome_References/Ensembl/drosophila_melanogaster/BDGP6.32/Release_105/GTF/Drosophila_melanogaster.BDGP6.32.105.gtf -config /public/singularity/containers/nextflow/lmb-nextflow/lmb.config --outdir results -dsl1 -bg

2. See error:

Execution completed unsuccessfully!

The exit status of the task that caused the workflow execution to fail was: 140.

The full error message was:

Error executing process > 'alevin_qc (SLX-21534.SITTG11.H37MYDRX2.s_1.r)'

Caused by:
  Process `alevin_qc (SLX-21534.SITTG11.H37MYDRX2.s_1.r)` terminated with an error exit status (140)

Command executed:

  mv 10x_V3_barcode_whitelist SLX-21534.SITTG11.H37MYDRX2.s_1.r_alevin_results/alevin/whitelist.txt
  alevin_qc.r SLX-21534.SITTG11.H37MYDRX2.s_1.r_alevin_results SLX-21534.SITTG11.H37MYDRX2.s_1.r SLX-21534.SITTG11.H37MYDRX2.s_1.r_alevin_results

Command exit status:
  140

Command output:
  (empty)

Command error:
  Loading required package: alevinQC
  Registered S3 method overwritten by 'GGally':
    method from   
    +.gg   ggplot2
  Loading required package: tximport
  Reading Alevin output files...
  Generating summary tables...
  Generating knee plot...

Work dir:
  /beegfs3/swingett/testing/scrna_seq/work/14/d79f9ee52236d2f715daa068a10c24

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`

The file .command.sh contains the text:

[swingett@hal d79f9ee52236d2f715daa068a10c24]$ cat .command.sh
#!/bin/bash -euo pipefail
mv 10x_V3_barcode_whitelist SLX-21534.SITTG11.H37MYDRX2.s_1.r_alevin_results/alevin/whitelist.txt
alevin_qc.r SLX-21534.SITTG11.H37MYDRX2.s_1.r_alevin_results SLX-21534.SITTG11.H37MYDRX2.s_1.r SLX-21534.SITTG11.H37MYDRX2.s_1.r_alevin_results

## Expected behaviour

A would have expected a QC plot to have been generated here.

## Log files

Have you provided the following extra information/files:

The log file is attached

launched the pipeline -->

## System

- Hardware: HPC
- Executor: slurm
- OS: Scientific Linux 7.9 (Nitrogen)
- Version 7.9

## Nextflow Installation

- Version: Nextflow version 22.03.1-edge build 5695

## Container engine

- Engine: Singularity
- version: scrnaseq 1.1.0
- Image tag: nfcore/scrnaseq:1.1.0

## Additional context

Also, on the page https://nf-co.re/scrnaseq/1.1.0/output the useful link "excellent tutorial" is broken: https://www.kallistobus.tools/getting_started.html

Thanks,

Steven

[nextflow.log](https://github.com/nf-core/scrnaseq/files/8541196/nextflow.log)

**Comment by apeltzer** [comment]
Thanks Steven for reporting - we are working on making all workflows run again in the `dev` release, though will also add alevin-fry here with some help from the authors. That will take a while, as we're all working on this part-time, please reach out on slack if you need to know more. 

Urls for the kallisto - bus tools were changed already yeah.

**Comment by StevenWingett** [comment]
Thanks for letting me know.  I see from the Slack chat that you are looking to incorporate Cell Ranger as well?  That will be an excellent addition to the pipeline.

All the best,

Steven

**Comment by apeltzer** [comment]
Yes, that is also planned and upgrading from alevin to alevin-fry, which is the "go to method" that succeeds alevin.

**Comment by apeltzer** [comment]
Please give `dev` or soon `2.0.0` a try - and open a new issue if that problem persists. It should work now, worst case we will fix it in 2.0.1 or similar.

**Comment by StevenWingett** [comment]
Thanks, I'll give this a try.

Best,
Steven
